apis/core/v1beta1/openstackcontrolplane_types.go

```diff
@@ -175,6 +179,25 @@ type OvnResources struct {
 	OVNNorthd ovnv1.OVNNorthdSpec `json:"ovnNorthd,omitempty"`
 }
 
+// OvsSection defines the desired state of OVS services
+type OvsSection struct {
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default=true
+	// Enabled - Whether OVS services should be deployed and managed
+	Enabled bool `json:"enabled,omitempty"`
+
+	// +kubebuilder:validation:Optional
+	// Template - Overrides to use when creating the OVN services
+	Template OvsResources `json:"template,omitempty"`
```

Comment:
can directly use Template ovsv1.OVSSpec `json:"template,omitempty"` to be consistent with other services(except ovn did a little differently as to manage both Northd and DBClusters).

Reply:
Done